content/docs/overview/observer/index.md

```diff
@@ -14,21 +14,22 @@ images: ["superficial-overview.png", "architecture.png", "object-hierarchy.png",
 
 {{< img-simple src="superficial-overview.png" alt="Overview" >}}
 
-Observer is a focuspoint for stats object origianted from different part of a WebRTC media services. 
+Observer consumes statistics generated from WebRTC end points like browsers and Selective Forwarding Units (SFUs).
```

Comment:
I copied all of these changes to the source page: https://github.com/ObserveRTC/observer/blob/master/docs/user-manual.md